libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -575,10 +417,15 @@ - (SFUserAccount *)applyCredentials:(SFOAuthCredentials*)credentials {
     if (currentAccount) {
         currentAccount.credentials = credentials;
     }else {
-        currentAccount = [self createUserAccountWithCredentials:credentials];
+        currentAccount = [[SFUserAccount alloc] initWithIdentifier:[self uniqueUserAccountIdentifier:credentials.clientId] clientId:credentials.clientId];
```

Comment:
Same here.  Let's just create a designated initializer for `SFUserAccount` that takes a credentials object.